packages/repository/test/acceptance/has-many.relation.acceptance.ts

```diff
@@ -76,6 +91,9 @@ describe('HasMany relation', () => {
       type: 'string',
     })
     name: string;
+
+    @hasMany({keyTo: 'customerId'})
```

Comment:
See https://github.com/strongloop/loopback-next/pull/1438#discussion_r198169162

Reply:
> Although I think that hasMany should have developed simultaneously with belongsTo (its counterpart), I think we have to stick to this manual providing of the foreign key since there is no belongsTo decorator to infer the data from. Thoughts?

I am arguing that we don't need `belongsTo` to infer the foreign key. We definitely don't need `belongsTo` relation in LB 3.x, see the following code snippet:

https://github.com/strongloop/loopback-datasource-juggler/blob/fa0039c6a8795d61c162c48160f03396b566cf44/lib/relation-definition.js#L678-L680

```js
  var relationName = params.as || i8n.camelize(modelTo.pluralModelName, true);
  var fk = params.foreignKey || i8n.camelize(thisClassName + '_id', true);
  var keyThrough = params.keyThrough || i8n.camelize(modelTo.modelName + '_id', true);
```

I am trying to say that we can use the same algorithm in LB4 too.

When `Category` is configured with `hasMany` relationship with `Product`, the foreign key `Product.prototype.categoryId` is called after the source model name.

Reply:
Ah, I see what you're saying. If this is how we do it in lb3, I'm ok with inferring the foreign key this way.